scripts/bytecodecompare/prepare_report.py

```diff
@@ -251,7 +251,7 @@ def detect_metadata_cli_option_support(compiler_path: Path):
             f"Compiler output:\n{process.stderr}\n",
             file=sys.stderr
         )
-        raise Exception("Failed to determine if the compiler supports the --metadata option.")
+        raise RuntimeError("Failed to determine if the compiler supports the --metadata option.")
```

Comment:
Not sure it's much better. It should have a proper exception type.

I guess we can leave it as is for now though.

Reply:
For sure. I just added as `RuntimeError` to fix the pylint. We should create proper exceptions for each case indeed.